hardcoded/hardcoded.go

```diff
@@ -1,17 +1,16 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Code generated for package hardcoded by go-bindata DO NOT EDIT. (@generated)
```

Comment:
Did the version of `go-bindata` change? I don't see a corresponding change in `go.mod`.